config/config.py

```diff
@@ -0,0 +1,187 @@
+import yaml, os
+
+class _DeskConfigDAO:
+	"""
+	Desk Config DAO
+	"""
+
+	KEY_ID = "id"
+	KEY_NAME = "name"
+	KEY_SENSOR_PIN = "sensor_pin"
+
+	def __init__(self, desk):
+		self.__desk_id = desk[self.KEY_ID]
+		self.__desk_name = desk[self.KEY_NAME]
+		self.__desk_pin = desk[self.KEY_SENSOR_PIN]
+
+	@property
+	def id(self):
+		return self.__desk_id
+
+	@property
+	def name(self):
+		return self.__desk_name
+
+	@property
+	def pin(self):
+		return self.__desk_pin
+
+
+class _DebugConfigDAO:
+	"""
+	Debug Config DAO
+	"""
+
+	KEY_ENABLED = "enabled"
+	KEY_REMOTE_SIMULATOR_CONFIG = "remote-simulator"
+
+	def __init__(self, debug):
+		self.__enabled = debug[self.KEY_ENABLED]
+		self.__remote_simulator_config = _RemoteSimulatorConfigDAO(debug[self.KEY_REMOTE_SIMULATOR_CONFIG])
+
+	@property
+	def enabled(self):
+		return self.__enabled
+
+	@property
+	def simulator_config(self):
+		return self.__remote_simulator_config
+
+
+class _RemoteSimulatorConfigDAO:
+	"""
+	Simulator Config DAO
+	"""
+	KEY_ENABLED = "enabled"
+	KEY_PORT = "port"
+
+	def __init__(self, simulator):
+		self.__enabled = simulator[self.KEY_ENABLED]
+		self.__port = simulator[self.KEY_PORT]
+
+	@property
+	def enabled(self):
+		return self.__enabled
+
+	@property
+	def port(self):
+		return self.__port
+
+
+class _ConfigDAO:
+	"""
+	Data Access Object (DAO) for the configuration
+	"""
+	KEY_DEBUG_CONFIG = "debug"
+	KEY_DESK_CONFIGS = "desks"
+
+	def __init__(self, loader):
+		self.__loader = loader
```

Comment:
Added this private variable so that we can call the `update_desk_list` from `YAMLConfigLoader`